app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
+
+/*navbar*/
+
+#headWrapper{
+  height: 60px;
+  border-bottom: 2px solid;
+  margin-bottom:30px;
```

Comment:
The margin here shouldn't be part of the top section (navigation bar), it should be part of the slides section. The reason for this is if you want to change the BG of the slide section, you will have a 30px white margin above it.